### PR TITLE
Add horizontal and vertical spacing tokens #345 #536

### LINKED ISSUE #345: Finalize Updated Grid Sizing Rules
From the board:

> Protocol needs either a standardized grid layout system (at most), or a predefined set number of container sizes (at least).

Also include advice on two-column layout. Per Craig Cook: 

> We’ve seen some designs placing content in two columns, which isn’t something we have a ready-made technique for. There are specific components that arrange side by side content, and we have some templates to arrange components in columns, but we don’t have a general-purpose, content-agnostic way to put things in columns. This is a common design pattern for which we should have a reusable template.

Need to define text line length for 1-column layout.

### LINKED ISSUE #536: Vertical Rhythm
Please use this template to request a new component.
If you're not sure whether a component already exists,
go ahead and file the issue here, and the team will triage appropriately.

Fill in as many sections as possible below.

### Please describe the use case(s) for this component

What does the component do? In what contexts is it used. Be as specific as possible.

### Please provide any designs or previous implementations of the proposed component

Add screenshots, mockups, code prototypes, etc. in this section.

## Please describe what this component does in the following conditions

* When set in front of a dark background
* When set in front of a light background
* When viewed in a mobile/responsive viewport

### Please provide examples of this type of component in use in non-Mozilla web properties

Add screenshots and links to web properties that use a similar component.
(We ask for this because we want to make sure we are following industry conventions while building components)

### Is the development of this component a blocking dependency for other work? Please explain if so

Generally speaking component definition should happen after a singleton instance has been developed,
or if the Protocol team has come to a consensus on need.

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Adds a spacing them with horizontal and vertical spacing. 
- Using the new spacing decreases outside padding on content containers at 1024px breakpoint (this is in preparation for updating components to use the wider spacing).
- Updates some hard-coded values to use existing variables

---

- ~I have documented this change in the design system.~
- [x] I have recorded this change in `CHANGELOG.md`.

### Issue

Fix #345
Fix #536

### Testing

Not much to see yet. You can compare http://localhost:3000/demos/content-containers.html to prod if you want.

### CODE REVIEW

File: src/assets/sass/protocol/includes/_themes.scss [hunk 1]
```diff
@@ -6,6 +6,34 @@
 // themes. Mixins and functions can draw from these variable maps, swapping to
 // a different map based on the value of the global $brand-theme variable.
 
+//* -------------------------------------------------------------------------- */
+// Vertical and Horizontal Spacing
+
+
+$theme-spacing-firefox: (
```
Comment (craigcook) [inline src/assets/sass/protocol/includes/_themes.scss:13]: Let's rename this `$theme-spacing` so it's not tied explicitly to the brand. 

In the near future we'll define a condensed spacing and probably give set it with a separate theme variable like we do with the condensed type scale, but for now just defining these units gives us a good start.

**Review by craigcook — CHANGES_REQUESTED** [review]
LGTM, let's just rename the map so it's distinct from the brand theme.

File: src/assets/sass/protocol/includes/_themes.scss [hunk 2]
```diff
@@ -6,6 +6,34 @@
...
+    v-grid-xs: $layout-md,
+    h-grid-xs: $layout-xs,
+
+    // sm is the same as xs
+    v-grid-sm: $layout-md,
+    h-grid-sm: $layout-xs,
+
+    // md
+    v-grid-md: $layout-md,
+    h-grid-md: $layout-md,
```
Comment (craigcook) [inline src/assets/sass/protocol/includes/_themes.scss:24]: Just noticed: Should this be `$layout-lg` (64px)? Referring to https://github.com/mozilla/protocol/issues/345#issuecomment-584856741

**Review by craigcook — CHANGES_REQUESTED** [review]

File: src/assets/sass/protocol/includes/_themes.scss (hunk 2 above)
Comment (stephaniehobson) [reply src/assets/sass/protocol/includes/_themes.scss:24]: Oh, yes.

**Comment by stephaniehobson** [comment]
Updated.

**Review by craigcook — APPROVED** [review]

**Pull request merged**